### ISSUE TITLE
User's menu suggestion

### ISSUE THREAD

**Issue by GijsTimmers** [zond/android-diplicity#62]
The current user's menu is very detailed, but it's pretty hard to read for non-developers.

I've made a quite ugly mockup for a new menu. This design should make it easier for users to find out the player's 'character'.
![layout](https://user-images.githubusercontent.com/972314/29190473-94545de2-7e1a-11e7-8c2b-fdaf2d6c10f8.png)

There are a couple of bars to show the player's rank compared to other players. This gives new users an idea of how good their opponent is.

Alternatively, a web like this would also be very interesting:
![89e9d8db75aa48e59cf7a6a68490c6b2--info-graphics-user-experience](https://user-images.githubusercontent.com/972314/29190513-ba9758b0-7e1a-11e7-82db-e08c9c5f51fc.jpg)

In this mockup, I've removed a couple of values that are not directly important to non-developers.

Also, I've replaced a few abstract scores by human-readable values.

This is just a first version and can still be improved.

**Comment by tttppp** [comment]
For reference I think this proposal currently skips the following information:
* Number of games in progress
* Dropped games (What are these?)
* Number of ready/active/NMR phases (although maybe this is shown being used to calculate the reliability percentage?)
* Existing reliability and quickness measures
* Ban information

It also adds the following information, which is currently untracked (as far as I know):
* Average player unreadiness period

I think this last information could be interesting to know, but so is the existing quickness measure, of whether they're ready before the deadline or not.

I like the star plot idea!

**Comment by zond** [comment]
I love that people want to improve the app!

Some answers:

- Dropped games are games where the player was NMR at the last phase.

- Yes, time before ready is untracked. Quickness is (played phases + 1) /
(ready phases + 1). Time before ready would be very skewed for players
playing mainly long/short games.

On Aug 10, 2017 23:04, "tttppp" <notifications@github.com> wrote:

For reference I think this proposal currently skips the following
information:

   - Number of games in progress
   - Dropped games (What are these?)
   - Number of ready/active/NMR phases (although maybe this is shown being
   used to calculate the reliability percentage?)
   - Existing reliability and quickness measures
   - Ban information

It also adds the following information, which is currently untracked (as
far as I know):

   - Average player unreadiness period

I think this last information could be interesting to know, but so is the
existing quickness measure, of whether they're ready before the deadline or
not.

I like the star plot idea!

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/zond/android-diplicity/issues/62#issuecomment-321672812>,
or mute the thread

**Comment by flurbius** [comment]
Player unreadiness could be recorded as a proportion of the phase length, in order to eliminate the skew.
A detailed user information page is a great idea, how about a page that compares 2 or 3 users?

**Comment by zond** [comment]
Yeah, unreadiness as a proportion of the phase length would work out well.

Comparison page would also be pretty cool!

**Comment by GijsTimmers** [comment]
Also, reliability could be expressed by a star rating or textual: "Very Poor", "Poor", "Moderate", "Good", "Excellent"
